packages/client/src/components/BigMovieCard/BigMovieCard.jsx

```diff
@@ -1,13 +1,156 @@
 import React from 'react';
-import { Container } from '@mui/material';
+import {
+  Card,
+  CardMedia,
+  CardContent,
+  Typography,
+  Box,
+  Divider,
+  Button,
+  Stack,
+} from '@mui/material';
+
 import RatingStars from '../RatingStars/RatingStars';
+import AddShoppingCartIcon from '@mui/icons-material/AddShoppingCart';
+import FavoriteIcon from '@mui/icons-material/FavoriteBorder';
 
-const BigMovieCard = ({ movieID }) => {
+const BigMovieCard = () => {
   return (
-    <Container sx={{ border: 1, width: 1518, height: 962 }}>
-      BigMovieCard
-      <RatingStars />
-    </Container>
+    <Card
+      sx={{
+        bgcolor: 'mainGreen',
+        display: 'flex',
+        justifyContent: 'center',
+        alignItems: 'center',
+        border: 1,
+        width: 1518,
+        height: 962,
+        paddingTop: 5,
+        paddingBottom: 5,
+        position: 'absolute',
+        left: 260,
+        top: 260,
+      }}
```

Comment:
I mentioned this in couple other PRs, but most of the components should be moved to some internal component library so you have one source of truth in most styling. You might want to discuss that in teams :)

For sure colors should be sourced in one file with constants, even if your colors are black, white and green. This makes it easier to make consistent changes across the project.